core/domain/story_services_test.py

```diff
@@ -142,7 +142,7 @@ def test_update_story_properties(self):
                 'cmd': story_domain.CMD_UPDATE_STORY_PROPERTY,
                 'property_name': story_domain.STORY_PROPERTY_THUMBNAIL_FILENAME,
                 'old_value': None,
-                'new_value': 'image.svg'
+                'new_value': 'test_svg.svg'
```

Comment:
```suggestion
                'new_value': 'image.svg'
```

Reply:
For the backend tests for `test_update_story_properties` to pass, try the following steps:

- Save a dummy image on the filesystem (which will be referenced by the `thumbnail_filename`) before calling the `story_services.update_story()` -- use this for reference: 

        # Save a dummy image on filesystem, to be used as thumbnail.
        with python_utils.open_file(
            os.path.join(feconf.TESTS_DATA_DIR, 'test_svg.svg'),
            'rb', encoding=None) as f:
            raw_image = f.read()
        fs = fs_domain.AbstractFileSystem(
            fs_domain.GcsFileSystem(
                feconf.ENTITY_TYPE_STORY, self.STORY_ID))
        fs.commit(
            '%s/image.svg' % (constants.ASSET_TYPE_THUMBNAIL), raw_image,
            mimetype='image/svg+xml')

Reply:
done